includes/login-register.php

```diff
@@ -64,63 +84,68 @@ function edd_register_form( $redirect = '' ) {
  * Process Login Form
  *
  * @since 1.0
+ * @since 2.9.24 No longer does validation which would prevent bruteforce detection plugins to be able to integrate.
+ *
  * @param array $data Data sent from the login form
  * @return void
 */
 function edd_process_login_form( $data ) {
+
 	if ( wp_verify_nonce( $data['edd_login_nonce'], 'edd-login-nonce' ) ) {
-		$user_data = get_user_by( 'login', $data['edd_user_login'] );
-		if ( ! $user_data ) {
-			$user_data = get_user_by( 'email', $data['edd_user_login'] );
-		}
-		if ( $user_data ) {
-			$user_ID = $user_data->ID;
-			$user_email = $user_data->user_email;
 
-			if ( wp_check_password( $data['edd_user_pass'], $user_data->user_pass, $user_data->ID ) ) {
+			$rememberme = isset( $data['rememberme'] ) ? (bool) $data['rememberme'] : false;
```

Comment:
```suggestion
			$rememberme = isset( $data['rememberme'] );
```

Reply:
This will provide `true` if `$data['rememberme']` is set to `false` since it will be set.

Reply:
@cklosowski `rememberme` won't be set at all if the checkbox isn't checked. This is the same as the expanded logic that existed previously as well:

```php
if ( isset( $data['rememberme'] ) ) {
	$data['rememberme'] = true;
} else {
	$data['rememberme'] = false;
}
```